languages/README.md

```diff
@@ -8,7 +8,15 @@ If you'd like to contribute a translation, please follow the Localizing section
 The generated POT template file is not included in this repository. To create this file locally, follow instructions from [README.md](https://github.com/woocommerce/woocommerce-admin/blob/master/README.md) to install the project, then run the following command:
 
 ```
-npm run build
+npm run i18n -- lang=xx_YY
```

Comment:
Small update here: the `--` can be removed.

Reply:
> I'm also having trouble loading the mo files

The mo files are created with the translation tool (poEdit, GlotPress, etc.).

Reply:
@joshuatf I removed the `--` and updated the instructions in `CONTRIBUTING.md` to be consistent.

Reply:
@rrennick Sorry if my issue wasn't clear.  I've already created the mo files and moved them to `wp-content/languages/plugins` per testing instructions but they don't seem to be having any effect.

When checking the loaded text domains using the plugin mentioned above, I don't see `woocommerce-admin` come up in the list.  Wondering if something is wrong with my environment or if you're able to recreate the issue.

Reply:
@joshuatf It is working for me
```
~/Sites/ron/wp-content/plugins/woocommerce-admin/languages
$ ls
README.md			woocommerce-admin-en_CA.po
~/Sites/ron/wp-content/plugins/woocommerce-admin/languages
$ ls ~/Sites/ron/wp-content/languages/plugins/woocommerce-admin*
/Sites/ron/wp-content/languages/plugins/woocommerce-admin-en_CA-2459417b12e8fe2d482313c6d8f9819a.json
/Sites/ron/wp-content/languages/plugins/woocommerce-admin-en_CA-27b105c531d6e08bd8855676f3a6164e.json
/Sites/ron/wp-content/languages/plugins/woocommerce-admin-en_CA-5c8a66f908738b94586c976c499edfe5.json
/Sites/ron/wp-content/languages/plugins/woocommerce-admin-en_CA-7bc4e52d9439cde288e4fdb2e16c0e61.json
/Sites/ron/wp-content/languages/plugins/woocommerce-admin-en_CA-946ffa241f2f7a67d85b6416fd6b9f50.json
/Sites/ron/wp-content/languages/plugins/woocommerce-admin-en_CA-bb2b82fbdf1b425e54cdec46f60a7f9b.json
/Sites/ron/wp-content/languages/plugins/woocommerce-admin-en_CA-cacb17be7bdb63bd3bb9c23d5a2305d7.json
/Sites/ron/wp-content/languages/plugins/woocommerce-admin-en_CA-d5abdf7f23e9521b725160dbd08295f1.json
/Sites/ron/wp-content/languages/plugins/woocommerce-admin-en_CA-eddaf6c0602832520b4958e0a7d25e59.json
/Sites/ron/wp-content/languages/plugins/woocommerce-admin-en_CA-f5fad241b4f471337325962233df4cd4.json
/Sites/ron/wp-content/languages/plugins/woocommerce-admin-en_CA.mo
```

<img width="701" alt="Screen Shot 2019-12-20 at 9 17 38 AM" src="https://user-images.githubusercontent.com/343847/71257250-9930a000-2309-11ea-9be4-c4d91361e75c.png">

While investigating this I found lines like
```
#: src/API/Leaderboards.php:321 client/analytics/report/downloads/config.js:49
```
in the .po file exported from GlotPress so I'll need to modify the replacement script to account for that patter.

Reply:
> I'll need to modify the replacement script to account for that pattern.

Done in `166294f6`